### PR TITLE
HTTP: Fixed missed parsing test

### PR DESCRIPTION
**Pull request by hongzhidao**
This issue was introduced in the commit 64934e5.

### CODE REVIEW

File: src/test/nxt_http_parse_test.c [hunk 1]
```diff
@@ -762,15 +762,14 @@ nxt_http_parse_test_request_line(nxt_http_request_parse_t *rp,
         return NXT_ERROR;
     }
 
-    if (rp->complex_target != (test->complex_target | test->quoted_target)) {
+    if (rp->complex_target != test->complex_target) {
```
Comment (ac000) [inline src/test/nxt_http_parse_test.c:765]: Not that it matters now, but the previous check looks suspect.

AIUI `->complex_target` is supposed to be treated as a boolean value so checking it against `->complex_target | ->quoted_target` looks wrong, although it happens to work (assuming the two values are either 0 or 1)...

**Comment by ac000** [comment]
You can get rid of the commit message and put instead...

```
Fixes: 64934e59f ("HTTP: Introduce quoted target marker in HTTP parsing")
Reviewed-by: Andrew Clayton <a.clayton@nginx.com>
```

Feel free to append your `Signed-off-by:` (once we've officially adopted the DCO this will become mandatory anyway)...

**Review by ac000 — APPROVED** [review]

File: src/test/nxt_http_parse_test.c (hunk 1 above)
Comment (hongzhidao) [reply src/test/nxt_http_parse_test.c:765]: Yep, it seems a tricky.

Comment (hongzhidao) [reply src/test/nxt_http_parse_test.c:765]: Btw, it looks like the CI test doesn't run with `./build/tests`. But BB did.

**Comment by ac000** [comment]
OK, so that went **completely wrong**!

What I suggested to do was this

```
commit 678c05686996e0aa6ddf79bf71279425306e9113
Author: Zhidao HONG <z.hong@f5.com>
Date:   Tue Apr 16 23:30:15 2024 +0800

    HTTP: Fixed missed parsing test

    Fixes: 64934e59f ("HTTP: Introduce quoted target marker in HTTP parsing")
    Reviewed-by: Andrew Clayton <a.clayton@nginx.com>
    Signed-off-by: Zhidao HONG <z.hong@f5.com>
```

What we ended up with is this

```
commit 678c05686996e0aa6ddf79bf71279425306e9113
Author: Zhidao HONG <z.hong@f5.com>
Date:   Tue Apr 16 23:30:15 2024 +0800

    Fixes: 64934e59f ("HTTP: Introduce quoted target marker in HTTP parsing")
    Reviewed-by: Andrew Clayton <a.clayton@nginx.com>
```

The Fixes tag has ended up as the **Subject** and there is **no** blank line separating it from the body.

Maybe I confused you when I said **message** when I obviously just meant the **body**,,,

Oh, well, that'll teach me **not** to approve things **before** I've seen the requested changes...

**Pull request merged**